src/lib/importSystem/record.ts

```diff
@@ -47,10 +47,17 @@ const getPriorFile = async (
     ],
     where: {
       importId,
-      status,
+      [Op.or]: [
+        { status },
+        {
+          status: IMPORT_STATUSES.COLLECTION_FAILED,
+          downloadAttempts: { [Op.gt]: 5 },
```

Comment:
I get why you're doing this, but I would think `getPriorFile` would return the prior file regardless of number of failed attempts 🤷 

What's the significance of 5? What does 5 translate to in terms of days?

Reply:
if its failed to download 5 times already, its not going to work. we were getting stuck trying to get the same file over and over